util/testutil/testing.go

```diff
@@ -154,3 +156,14 @@ func formatMessage(msgAndArgs []interface{}) string {
 	}
 	return ""
 }
+
+// TolerantVerifyLeak verifies go leaks but excludes the go routines that are
+// launched as side effects of some of our dependencies.
+func TolerantVerifyLeak(m *testing.M) {
```

Comment:
mhmhm, let's remove this function and add those explicitly so we are aware. This should go at some point as issue to such projects :hugs:

Reply:
Well they will be the same in a *lot* of prometheus packages.

Reply:
mhmhmh, fair